tox.ini

```diff
@@ -1,8 +1,8 @@
 [pytest]
 addopts = -v -rxs -s --color=yes --tb=native --ignore=setup.py --doctest-modules --doctest-glob 'tests/**/*.txt' --cov-report term-missing --cov owslib
 norecursedirs = .git docs examples etc cov* *.egg* pytest* .tox
-python_files=check_*.py
-python_functions=check
+#python_files=check_*.py
+#python_functions=check
```

Comment:
I would remove rather than comment this out - as there are no functions/files using the check_ syntax

Reply:
@johanvdw 
I have cleaned up `tox.ini` as requested in the PR.